build.gradle

```diff
@@ -13,7 +13,7 @@ buildscript {
 
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.3'
```

Comment:
Please explain for this update.

Reply:
Gradle 3.6.0 fixes various known issues 3.5.3, e.g. linting is slow, missing manifest class, etc. Is there a strong reason not to upgrade this dependency?

Reply:
Okay, great.